.github/workflows/build_and_deploy_hold.yml

```diff
@@ -16,16 +16,42 @@ jobs:
     with:
       cloud_region: EU
 
+  merge_multiple_us_artifacts:
+    runs-on: ubuntu-latest
+    needs: call_build
+    steps:
+      - name: Merge into build-output-US Artifact
+        uses: actions/upload-artifact/merge@v4
+        with:
+          name: build-output-US
+          pattern: build-output-US-*
+          merge-multiple: true
+
+  merge_multiple_eu_artifacts:
```

Comment:
seeing some warnings [here](https://github.com/jesuyedavid/answers-search-ui/actions/runs/8993739075) from these merge jobs when they ran in the forked repo

Reply:
looks like the warning is [expected](https://github.com/octokit/request-action/issues/26#issuecomment-661936263). i can make a follow up item to address this warning specifically: a potential solution is to use github secrets instead.